src/hotspot/share/gc/parallel/parallelInitLogger.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#include "precompiled.hpp"
+#include "gc/parallel/parallelInitLogger.hpp"
+#include "gc/shared/genArguments.hpp"
+#include "gc/shared/gcLogPrecious.hpp"
+
+void ParallelInitLogger::print_gc_specific() {
+  log_info_p(gc, init)("Alignments:"
+                     " space_align=" SIZE_FORMAT "%s"
+                     " gen_align=" SIZE_FORMAT "%s"
+                     " heap_align=" SIZE_FORMAT "%s",
+                     byte_size_in_exact_unit(SpaceAlignment), exact_unit_for_byte_size(SpaceAlignment),
+                     byte_size_in_exact_unit(GenAlignment), exact_unit_for_byte_size(GenAlignment),
+                     byte_size_in_exact_unit(HeapAlignment), exact_unit_for_byte_size(HeapAlignment)
+                     );
```

Comment:
The indentation here got a bit of now. I also took a look at the current output and we do not use '=' anywhere else so I suggest going with something like this:
```suggestion
  log_info_p(gc, init)("Alignments:"
                       " Space " SIZE_FORMAT "%s,"
                       " Generation " SIZE_FORMAT "%s,"
                       " Heap " SIZE_FORMAT "%s",
                       byte_size_in_exact_unit(SpaceAlignment), exact_unit_for_byte_size(SpaceAlignment),
                       byte_size_in_exact_unit(GenAlignment), exact_unit_for_byte_size(GenAlignment),
                       byte_size_in_exact_unit(HeapAlignment), exact_unit_for_byte_size(HeapAlignment));
```
This would generate output like this:
```
[0.811s][info][gc,init] Compressed Oops: Enabled (Zero based)                                                                                                                                                                                                                                                        
[0.811s][info][gc,init] Alignments: Space 512K, Generation 512K, Heap 2M                                                                                                                                                                                                                                             
[0.811s][info][gc,init] Heap Min Capacity: 8M                                                                                                                                                                                                                                                                        
[0.811s][info][gc,init] Heap Initial Capacity: 1002M                                                                                                                                                                                                                                                                 
[0.811s][info][gc,init] Heap Max Capacity: 16012M                                                                                                                                                                                                                                                                    
[0.811s][info][gc,init] Pre-touch: Disabled                                                                                                                                                                                                                                                                          
```

Reply:
Yes, looks much better! I just took the format from the original pagesize output, but this looks better.